.github/workflows/build_and_deploy.yml

```diff
@@ -57,7 +57,7 @@ jobs:
         shell: bash
         if: github.event.number != ''
         run: |
-          az aks get-credentials -g s189t01-tsc-ts-rg -n s189t01-tsc-test-aks
+          make review get-cluster-credentials
```

Comment:
Always use `ci` in the pipeline:
make ci review get-cluster-credentials

Reply:
done